src/components/BackstageComparison/FeatureComparisonTable.js

```diff
@@ -0,0 +1,413 @@
+import React, { Fragment } from 'react';
+import { CheckIcon, MinusIcon } from '@heroicons/react/solid';
+import { TextLink } from 'components';
+
+import COMPETITORS from './competitors';
+
+const sections = [
+  {
+    name: 'Software catalog',
+    features: [
+      {
+        name: 'Bring your own data model',
+        competitor: { Roadie: 'Yes', Backstage: 'Yes, with some effort' },
+      },
+      {
+        name: 'Entity definition',
+        competitor: { Roadie: 'Markdown, UI, or via API', Backstage: 'Markdown' },
+      },
+      {
+        name: 'API-based ingestion and update',
+        competitor: { Roadie: true, Backstage: false },
+      },
+      {
+        name: 'Out of the box data ingestion (k8s, github, CI/CD, etc)',
```

Comment:
How come we're comfortable giving Backstage a `yes` and Port are not?

Reply:
For which section?

Reply:
"Out of the box data ingestion"